lib/inv_eigcg_quda.cpp

```diff
@@ -21,189 +21,209 @@
 
 #include <deflation.h>
 
-
 /*
-Based on  eigCG(nev, m) algorithm:
-A. Stathopolous and K. Orginos, arXiv:0707.0131
+  Based on  eigCG(nev, m) algorithm:
+  A. Stathopolous and K. Orginos, arXiv:0707.0131
 */
 
 namespace quda {
 
-   using namespace blas;
-   using namespace Eigen;
-
-   using DynamicStride   = Stride<Dynamic, Dynamic>;
-   using DenseMatrix     = MatrixXcd;
-   using VectorSet       = MatrixXcd;
-   using Vector          = VectorXcd;
-   using RealVector      = VectorXd;
-
-//special types needed for compatibility with QUDA blas:
-   using RowMajorDenseMatrix = Matrix<Complex, Dynamic, Dynamic, RowMajor>;
-
-   static int max_eigcg_cycles = 4;//how many eigcg cycles do we allow?
-
-
-   enum  class libtype {eigen_lib, magma_lib, lapack_lib, mkl_lib};
-
-   class EigCGArgs{
+  using namespace blas;
```

Comment:
This file, inv_eigcg_quda.cpp has lots of changes here, but this pull isn't really focused on EigCG.  What is being changed in the algorithm here?

Reply:
I think I made some attempts and integrating the file, and clang formatting changed a lot of stuff. I'm going to checkout the develop version.